test/ToucanRegenBridge.js

```diff
@@ -33,15 +35,17 @@ describe("Bridge contract", function () {
 	});
 
 	beforeEach(async function () {
-		[admin, bridgeAdmin, broker] = await ethers.getSigners();
+		[admin, tokenIssuer, broker] = await ethers.getSigners();
 
 		// Deploy ToucanRegenBridge
-		bridge = await deployBridge(bridgeAdmin.address, nctPool.address);
+		bridge = await deployBridge(admin, nctPool.address);
+		await bridge.connect(admin).grantRole(keccak256(toUtf8Bytes("TOKEN_ISSUER_ROLE")), tokenIssuer.address);
 		await tco2Factory.addToAllowedBridges(bridge.address);
 	});
 
-	it("Should set the right owner and initial parameters", async function () {
-		expect(await bridge.owner()).to.equal(admin.address);
+	it.skip("Should set the right default admin and initial parameters", async function () {
+		const isAdmin = await bridge.hasRole(keccak256(toUtf8Bytes("DEFAULT_ADMIN_ROLE")), admin.address);
```

Comment:
No, this is wrong.  `DEFAULT_ADMIN_ROLE` is the zero value.  Put this after the imports:

```typescript
    const DEFAULT_ADMIN_ROLE = ethers.constants.HashZero;
```

then

```suggestion
		const isAdmin = await bridge.hasRole(DEFAULT_ADMIN_ROLE, admin.address);
```